### PR TITLE
Adding js feature

### PR DESCRIPTION
**Pull request by MenendezON**
This pull request concerns the implementation of javascript in our project. By using js to split out the menu.

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where ESLint check is not running. Kindly use the `linters.yml` file from the [HTML-CSS-JS](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) folder of the llinters config repository.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: public/style.css [hunk 1]
```diff
@@ -123,17 +124,28 @@ ul {
...
 }
 
+.nav div {
+  padding: 15px;
+  font-size: 1.7em;
+  color: var(--white);
+  text-align: right;
+  margin-right: 15px;
+  cursor: pointer;
+}
```
Comment (Ibizugbe) [inline public/style.css]: Nice job implementing the functionalities of the mobile menu👍. Let's try to make the mobile menu look more like the design as we ought to stick to the design in this milestone👍

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

Kindly make a comparison and make the necessary changes🥲 Take note of the blur property on the design and the font property of your texts👍.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅
- [x] Linters are properly setup and they are passing👍.
- [x] The functionalities of the mobile menu have been implemented👍.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] Kindly take note that your entire project is looking a bit different from the given design take note of the positioning of differenrt elements and font properties and the position of your texts and try to fix them as sticking to the given design is a good display of professionalism😉👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @MenendezON, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested changes implemented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**